bin/topological-inventory-inventory-upload

```diff
@@ -15,8 +15,12 @@ def parse_args
         :default => ENV.fetch("QUEUE_HOST", "localhost")
     opt :queue_port, "Port of the Platform's kafka queue", :type => :int,
         :default => (ENV.fetch("QUEUE_PORT", 9092)).to_i, :required => false
-    opt :ingress_api, "Hostname of the ingress-api route", :type => :string,
-        :default => ENV.fetch("INGRESS_API", "http://localhost:9292")
```

Comment:
Before this change this would default to `http://localhost:9292` do we want to try to preserve that behavior for running locally?